resource/css/_page_list.scss

```diff
@@ -14,6 +14,9 @@
   }
 
   .page-list-ul {
+    width: calc(100vw - 2em);
```

Comment:
Because of `.page-list-ul` is also used in the search result, it should be removed.

Reply:
fixed the style and also adjust adjust small device style.